bin/node/runtime/src/lib.rs

```diff
@@ -91,7 +91,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 247,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
```suggestion
	impl_version: 0,
```

It does not changes any implementation^^

Reply:
Our CI seems to require it though -- https://github.com/paritytech/substrate/blob/master/.maintain/gitlab/check_runtime.sh#L40
Any change in `frame` has to at least bump `impl_version`, if I understand the script correctly.

Reply:
this check can be ignored IFAIK.